gkTally/profitLoss.py

```diff
@@ -21,68 +21,70 @@ def parse_args():
 
 # string containing key words to decide if a transaction is to be considered for
 # profit loss calculation
-profit_loss = ["bought", "sold"]
-investment = ["funding", "wire"]
+profitLossIncludeList = ["bought", "sold"]
+# exclude list - transations that needs to be excluded
+excludeList = ["Bought 100 AAPL"]
+# Investment List - transactions that needs to be considered as investment
+investmentList = ["funding", "wire"]
 
 
-class transaction_finder:
+class transactionFinder:
     def __init__(self, in_file):
         self.in_f = in_file
-        self.profit_l = 0.0
-        self.first_date = ""
-        self.total_inv = 0.0
+        self.profitL = 0.0
+        self.firstDate = ""
+        self.totalInv = 0.0
 
-    def str_find(self, s, key):
+    def strFind(self, s, key):
         for i in key:
-            if ((s.lower()).find(i)) != -1:
```

Comment:
We dont need this if. Only one if is enough. I am sure you can do it with list comprehension and remove this function all together. If you can create issue to track these comments, it would be great.